cmd/contour/servecontext.go

```diff
@@ -131,6 +132,14 @@ type serveContext struct {
 
 	// Name of the envoy service to inspect for Ingress status details.
 	EnvoyServiceName string `yaml:"envoy-service-name,omitempty"`
+
+	// DefaultHTTPVersions defines the default set of HTTPS
+	// versions the proxy should accept. HTTP versions are
+	// strings of the form "HTTP/xx". Supported versions are
+	// "HTTP/1.1" and "HTTP/2".
```

Comment:
Do we need to include `HTTP` in the value? The parameter name specifies it can only be HTTP.

Reply:
It's not strictly necessary, but IMHO it reads better and makes the config more obvious :)